designs/virtualization/4.5/Templates/Templates.md

```diff
@@ -0,0 +1,29 @@
+---
+parent: Virtualization
+version: 4.5
+---
+
+## Virtual machines and VM templates are located under Virtualization
+
+One of the options for users to create a VM is when they choose a to create it from a template.
+Under the left side navigation tab ‘Virtualization’ we'll have 2 tabs: ‘Virtual Machine Templates’ and ‘Virtual Machines’.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual machine templates tab where they can see the full list of templates.
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
+
+Both options will lead the user to the general step of the wizard where they will need to provide a name for the VM they want to create.
+
+![UserNeedsToProvideName](img/provide-name.png)
```

Comment:
Ya we had designed it that way but this is how it was implemented. I actually think the import ended up the same way :( I took screen shots for this. I can see if I can get some screenshots for the import too b/c there are small changes there too. I'm always a bit torn to show what was implemented vs what was designed.

Reply:
@matthewcarleton I tend to think we should show the intended design (to a certain degree) and file bugs for things that we know we want changed if they go against our conventions. I say to a certain degree only because where this logic doesn't hold up as much is with PF stuff. I think we want things to be all PF4 eventually but the reality is, it will take some time to convert pieces so for PF cases I tend to think we go more with not necessarily just what's implemented but at least what is our current OpenShift convention. What do you think?

Reply:
+1 I have been pushing on this from the PF side to get some solid direction with alerts and where they should be placed consistently. I believe the consensus was that they belong at the bottom - so I can update the other designs to reflect this.

Reply:
just thinking about this a bit more too @lizsurette @beanh66 
The top level alert is really about notifying the user of all the different things that need to be addressed to enable the import. It's nice to have at the top b/c the user sees it right away and can use it to navigate around to the different steps that require attention. 
The bottom alert would be more about the specific step and notifying the user of required fields on that step if they are trying to go to the next step without dealing with them first. 
This design can be adjust a bit to show this better but I really want to hear what you both think first.

Reply:
I guess the question would be... are they taken to the top of the form on fail? If they stay here at the bottom that makes sense to keep it at the bottom to show them in context what the error is. Do I dare suggest it's a question that should go back to PatternFly around wizard?

Reply:
ya - I was on a call with PF that was just about this and I think they settled on the error being at the bottom of the form. Good a11y seems like it wouldn't move the user but alert them where they are. They would also get the inline input errors as well. The top level error only surfaces errors at the step level and it's persistent so it's a bit different.